src/sprites.asm

```diff
@@ -258,14 +258,14 @@ DEC $00
 CLC
 ADC $E4,X
 STA $E4,X
-; PHP
-; PHA
-; SEC
-; SBC $1534,X
-; STA $1528,X
-; PLA
-; STA $1534,X
-; PLP
+PHP
+PHA
+SEC
+SBC $1534,X
+STA $1528,X
+PLA
+STA $1534,X
+PLP
```

Comment:
This should stay commented out. This snippet of code is copied from the original rotating platform code, and this is the part which is actually responsible for updating $1528,x (the platform's x movement against the previous frame) and $1534,x (the x-position of the tip of the platform). Uncommenting it _technically_ doesn't do any harm, but it's not necessary since the computation has already been done this frame.